README.md

```diff
@@ -1,5 +1,5 @@
 
-![Go](https://github.com/rsdmike/app-functions-sdk-go/workflows/Go/badge.svg)
+[![Build Status](https://jenkins.edgexfoundry.org/view/EdgeX%20Foundry%20Project/job/edgexfoundry/job/app-functions-sdk-go/job/master/badge/icon)](https://jenkins.edgexfoundry.org/view/EdgeX%20Foundry%20Project/job/edgexfoundry/job/app-functions-sdk-go/job/master/) [![Code Coverage](https://codecov.io/gh/edgexfoundry/app-functions-sdk-go/branch/master/graph/badge.svg?token=E4uzIvukPu)](https://codecov.io/gh/edgexfoundry/app-functions-sdk-go) [![Go Report Card](https://goreportcard.com/badge/github.com/edgexfoundry/app-functions-sdk-go)](https://goreportcard.com/report/github.com/edgexfoundry/app-functions-sdk-go) [![GitHub Latest Dev Tag)](https://img.shields.io/github/v/tag/edgexfoundry/app-functions-sdk-go?include_prereleases&sort=semver&label=latest-dev)](https://github.com/edgexfoundry/app-functions-sdk-go/tags) ![GitHub Latest Stable Tag)](https://img.shields.io/github/v/tag/edgexfoundry/app-functions-sdk-go?sort=semver&label=latest-stable) [![GitHub License](https://img.shields.io/github/license/edgexfoundry/app-functions-sdk-go)](https://choosealicense.com/licenses/apache-2.0/) ![GitHub go.mod Go version](https://img.shields.io/github/go-mod/go-version/edgexfoundry/app-functions-sdk-go) [![GitHub Pull Requests](https://img.shields.io/github/issues-pr-raw/edgexfoundry/app-functions-sdk-go)](https://github.com/edgexfoundry/app-functions-sdk-go/pulls) [![GitHub Contributors](https://img.shields.io/github/contributors/edgexfoundry/app-functions-sdk-go)](https://github.com/edgexfoundry/app-functions-sdk-go/contributors) [![GitHub Committers](https://img.shields.io/badge/team-committers-green)](https://github.com/orgs/edgexfoundry/teams/app-functions-sdk-go-committers/members) [![GitHub Commit Activity](https://img.shields.io/github/commit-activity/m/edgexfoundry/app-functions-sdk-go)](https://github.com/edgexfoundry/app-functions-sdk-go/commits) ![Go](https://github.com/rsdmike/app-functions-sdk-go/workflows/Go/badge.svg)
```

Comment:
Should we drop this badge? https://github.com/rsdmike/app-functions-sdk-go/workflows/Go/badge.svg
![image](https://user-images.githubusercontent.com/1890273/115290448-661e3c80-a108-11eb-99a3-b205053eff0c.png)

Reply:
@ernestojeda it appears the respective GitHub worfklow is still defined thus the reason I left this badge alone.

Reply:
@ernestojeda I updated the GitHub Actions Workflow badge to point to the correct place.

@lenny-intel This repo is also building using GitHub Actions Workflow and there was a previous badge in the readme for it, do you want to keep the badge?  Note, I did update it to reference the correct location (it was pointing to rsdmike personal repo before).

Reply:
Looks good. THX.